src/Fantomas/CodePrinter.fs

```diff
@@ -730,11 +735,22 @@ and genPropertyWithGetSet astContext (b1, b2) =
         +> unindent
     | _ -> sepNone
 
-and genPropertyKeyword (pkw: PropertyKeyword option) (ctx: Context) =
-    match pkw with
+/// <summary>Generate the keyword <code>and</code> or <code>with</code>, along with any matching syntax trivia, for a given keyword</summary>
```

Comment:
Now you are just making all the rest look bad with such a nice triple-slash comment 🙈.

Reply:
I commented the heck out of it because it was a fairly subtle interaction that I knew someone would have questions about later on :) it's self-interest only, never fear!

Reply:
Yup, most likely future me included 🙋‍♂️.
To avoid any confusion: please leave it in, it is beautiful!